src/Illuminate/Cache/RateLimiting/Limit.php

```diff
@@ -53,9 +53,9 @@ public function __construct($key = '', int $maxAttempts = 60, int $decayMinutes
      * @param  int  $maxAttempts
      * @return static
      */
-    public static function perMinute($maxAttempts)
+    public static function perMinute($maxAttempts, $decayMinutes = 1)
```

Comment:
This is also a breaking change and best to sent to master.

Reply:
I'm sure that I'm missing something, but I don't see how this is a breaking change? All existing code to `perMinute` would still work properly, since the second argument is optional, wouldn't it?

Reply:
If someone if overwriting this method their implementation will break.

Reply:
Aaah, OK thanks. That makes sense.

Reply:
Thoughts on whether it would make more sense to create a new `perMinutes` or `perCustom` method instead? That wouldn't be a breaking change.